sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/azure_appconfiguration_requests.py

```diff
@@ -10,12 +10,16 @@
 from azure.core.pipeline.policies import HTTPPolicy
 from .utils import parse_connection_string, get_current_utc_time
 
-class AppConfigConnectionStringCredential():
+
+class AppConfigConnectionStringCredential:
```

Comment:
This looks like an odd module to add the credential class to. Is there a specific reason this was chosen?

Reply:
...and was it black that "fixed" this to make it an old style class for Python2?

Reply:
> ...and was it black that "fixed" this to make it an old style class for Python2? If so, that is a new (and, for us, unfortunate) feature in black.

Yes. Black "fixed" this.

Reply:
Black only did that because there was not parent class inside the parens. I haven't encountered that style before.

Reply:
Ah, I was so focused on what it changed it **to** that I didn't see that it was already incorrect in the **from** :). We should have `object` as the base class. Once we are in a position to stop supporting Python2, we can fix this up and life will be better.